packages/devtools_shared/lib/src/memory/memory_json.dart

```diff
@@ -231,8 +234,8 @@ class AllocationMemoryJson extends MemoryJson<ClassHeapDetailStats> {
   /// Given a JSON string representing an array of HeapSample, decode to a
   /// List of HeapSample.
   AllocationMemoryJson.decode({
-    String argJsonString,
-    Map<String, dynamic> argDecodedMap,
+    String? argJsonString,
```

Comment:
this should probably be non-nullable and required as there isn't much point in trying to decode without specifying what you are decoding. I'm assuming that you lways need to pass argJsonString and that passing in just argDecodedMap isn't required.

Reply:
done